src/ruby/ext/grpc/rb_channel_credentials.c

```diff
@@ -138,10 +138,14 @@ static ID id_pem_cert_chain;
     ...
     creds3 = Credentials.new(pem_root_certs, pem_private_key,
                              pem_cert_chain)
+    ...
+    creds4 = Credentials.new(fallback_creds)
     pem_root_certs: (optional) PEM encoding of the server root certificate
     pem_private_key: (optional) PEM encoding of the client's private key
     pem_cert_chain: (optional) PEM encoding of the client's cert chain
-    Initializes Credential instances. */
+    Initializes Credential instances.
+    fallback_creds: (optional ChannelCredentials) fallback credentials to create
+    XDS credentials. */
```

Comment:
Reorder the lines as follows:
```
    fallback_creds: (optional ChannelCredentials) fallback credentials to create
    XDS credentials.
    Initializes Credential instances. */
```
So all the options are described in one place and the last line explains what the function does.

Reply:
comment addressed